lib/rouge/lexers/cython.rb

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    load_lexer 'python.rb'
+
+    class Cython < Python
+      title "Cython"
+      desc "Cython and Pyrex source code (cython.org)"
+      tag 'cython'
+      aliases 'pyx', 'pyrex'
+      filenames '*.pyx', '*.pxd', '*.pxi'
+      mimetypes 'text/x-cython', 'application/x-cython'
+
+      def self.keywords
+        @keywords ||= super + Array.new(%w(
```

Comment:
The `%w()` literal produces an array so need to wrap that in an array.

Reply:
Fixed this by removing the `Array.new` call.